docs/timelines/creating-a-timeline.md

```diff
@@ -0,0 +1,103 @@
+---
+id: creating-a-timeline
+title: Creating a Timeline
+sidebar_label: Creating a Timeline
+---
+
+You can create a timeline by using Live Record mode, by entering triggers manually at specified times, or a combination of the two.
+
+
+
+Programming a Timeline using Live Record
+----------------------
+
+Live record is useful to capture a busked show, where you are operating the playbacks manually in time with music or other live cues. You can do a live record multiple times on the same timeline to build up a more complex show.
+
+>  Recall of Palettes will not be recorded using timeline Live Record. You need to save looks to a playback to use them on a timeline.
+
+This is how you program a timeline using Live Record, from playbacks which you have previously programmed:
+
+1. If you want to use internal timecode for recording rather than feeding in a timecode source, ensure the "Timecode Disconnect" \{Link\} button is selected to show a broken link.
+
+2. Press \<Record\>, \[Timeline\], then the **Select button** of the fader where you want to store the timeline. The Timeline View window will automatically open.
+
+3. Press the \{O\} (record) button in the top left of the Timeline View window. 
+
+4. Select which track on the timeline you want to record onto using the \[Track\] softkey or tap on the flashing dotted area of the track you want to use. 
```

Comment:
I think you can press record again to immediately enter live record with the current track selection